### DISCUSSION TITLE
[NFR]: In my project, i need to create a okta authentication. I can able to do so with uri based. My Question is, is there any dedicated library to implement it without any uri based request? Iam using phalcon 3 for my project.

### DISCUSSION THREAD

**Discussion by Vignesh7198** [phalcon/cphalcon#16294] [Q&A]
.

**Comment by Jeckerson** [comment]
I think there are none, you will need to implement new one or use some generic (if exists) that in composer package manager universe.

Also, I suggest to migrate to the newer version of Phalcon, as existing libraries might require latest & supported PHP version(s).
